api/openapi/authentication.md

```diff
@@ -20,6 +20,8 @@ curl --header "Authorization: Bearer 1a6a9936ad150a2ee345c65331da7a3ccc2de" http
 
 By default, API keys are only generated for admins, but you can easily customize Solidus to generate them for all users, which is useful for instance if you want users to be able to sign in and manage their profile via the API.
 
+The `API key` is mandatory for each endpoint by default. You can change this configuration [here](https://github.com/solidusio/solidus/blob/2b79f72aa53f5caa850c587888fff46c1c91f7b7/api/lib/spree/api_configuration.rb#L5) if you want to avoid the default behaviour exposing some endpoints without an API key. An example is the [GET product list](https://solidus.stoplight.io/docs/solidus/08307f3d809e7-list-products).
```

Comment:
I think that the current wording is a bit unclear (I had to read it twice to understand). What about 👇 ?

```suggestion
The `API key` is mandatory for each endpoint by default. You can change this configuration [here](https://github.com/solidusio/solidus/blob/2b79f72aa53f5caa850c587888fff46c1c91f7b7/api/lib/spree/api_configuration.rb#L5) to avoid the default behavior and expose some endpoints without an API key. An example could be the [GET product list](https://solidus.stoplight.io/docs/solidus/08307f3d809e7-list-products) endpoint.
```

Also, thoughts on changing "here" with "with the `Spree::Api::Config.requires_authentication` preference"?

Reply:
Thank you for the suggestion, I think your copy is better and I want to change it. About the link against the `Spree::Api::Config.requires_authentication`, I prefer the link or both to help the user to understand better. What do you think?